lib/engine.py

```diff
@@ -476,6 +475,25 @@ def exists(self, script):
             )
             for key in list(script.urls.keys()) if key])
 
+    def to_csv(self):
+        from retriever.lib.tools import sort_csv
+        if self.name is "sqlite":
+            self.connection.text_factory = str
+        csvfile_output = (self.table_name() + '.csv')
+        self.get_cursor()
+        self.cursor.execute("SELECT * FROM " + self.table_name() + ";")
+        row = self.cursor.fetchone()
```

Comment:
Seems like it would be more readable to move this down to be the line right before the `while` loop starts.